config/bidderinfo.go

```diff
@@ -197,7 +197,8 @@ func LoadBidderInfo(path string) (BidderInfos, error) {
 
 		bidderName := strings.Split(fileName, ".")
 		if len(bidderName) == 2 && bidderName[1] == "yaml" {
-			infos[(bidderName[0])] = info
+			// store all bidder names loaded from file system in lowercase
```

Comment:
Nitpick: Please expand on the comment to include this is mention the conversion to lower case is to match the same behavior of config.BidderInfos .